docs/getting-started/configuration.md

```
@@ -1,325 +1,248 @@
 
-```yaml
-rules:
-  - description: All pull requests must have a min num of approvals unless the author is a maintainer
-    enabled: true
-    severity: high
-    event_types: [pull_request]
-    parameters:
-      min_approvals: 2
-```
+| Field | Required | Description |
+|-------|----------|-------------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)